test/README

```diff
@@ -9,3 +9,11 @@ symlinked from test/output/latest/html/index.html.
 
 *--output-dir*
 	Litani will write all of its test output files for this run to this path
+
+*--fast*
+    Do not run tests with SLOW variable set to True
+
+## Writing Tests
+
+Each test must set a variable called SLOW.
+If the run time of a test is >= 10 seconds, SLOW should be set to True, and False otherwise
```

Comment:
Should this be updated to reflect what you have mentioned in the PR's description?